storage/azure/src/frontend/AzureFrontendStorage.ts

```diff
@@ -35,22 +32,21 @@ export class AzureFrontendStorage extends FrontendStorage {
     input: (FrontendUrlDownloadInput | FrontendAzureConfigDownloadInput) & {
       transferType: "buffer";
     }
-  ): Promise<Buffer>;
+  ): Promise<ArrayBuffer>;
 
   public download(
     input: (FrontendUrlDownloadInput | FrontendAzureConfigDownloadInput) & {
       transferType: "stream";
     }
-  ): Promise<Readable>;
+  ): Promise<ReadableStream>;
 
   public async download(
     input: FrontendUrlDownloadInput | FrontendAzureConfigDownloadInput
   ): Promise<FrontendTransferData> {
-    const downloadStream = await this._clientWrapperFactory
+    const downloadBlob = await this._clientWrapperFactory
       .create(input)
       .download();
-
-    return streamToTransferTypeFrontend(downloadStream, input.transferType);
+    return downloadBlob.arrayBuffer();
```

Comment:
Looks like AzureFrontendStorage is missing implementation for download&transferType="stream". should I just convert it to stream so it can pass the tests?